docs/platform/howto/byoc/create-custom-cloud.rst

```diff
@@ -0,0 +1,137 @@
+
+In the **Create custom cloud** wizard, proceed as follows:
+
+1. Specify the following:
+
+   * Custom cloud name
+   * Cloud provider
+
+     .. important::
+
+        **Amazon Web Services (AWS)** is the only option supported currently.
+
+   * Region
+   * CIDR
+
+     Aiven needs CIDR for the `CIDR block of the VPC <https://docs.aws.amazon.com/vpc/latest/userguide/vpc-cidr-blocks.html>`_ that will be created in your AWS account.
+
+     * Specify inbound rules with the CIDR block notation, for example: 200.1.2.3/32 (allowing 200.1.2.3 as a single address), 0.0.0.0/0 (allowing traffic from anywhere), or 100.1.0.0/16 (allowing traffic from 100.1..).
+     * To create VPC peerings with that VPC, choose a CIDR block that doesn't overlap with CIDR blocks of peer VPCs.
+     * Keep in mind that CIDR block needs be large enough so that, after splitting it into per-region subnets, each subnet has enough addresses to fit required services.
+
+   * :ref:`Deployment model <byoc-deployment>`
+
+     The deployment model determines how resources within your Aiven organization are arranged. It also imposes the method of connectivity between Aiven's control plane and networks under your cloud provider account.
+
+     .. important::
+
+        **BYOC standard** is the only option supported currently.
+
+1. Select **Next**.
+   
+.. topic:: Result
+
+    Your IaC Terraform template gets generated based on your inputs. You can view, copy, or download it. Now, you can use the template to :ref:`acquire Role ARN <acquire-role-arn>`.
+
+.. _acquire-role-arn:
+
+Deploy the template to acquire ``Role ARN``
+'''''''''''''''''''''''''''''''''''''''''''
+
+Role ARN is an `identifier of the role <https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html>`_ created when running the infrastructure template in your AWS account. Aiven uses Role ARN to `assume the role <https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html>`_ and run operations such as creating VMs for service nodes in your BYOC account.
+
+Use the Terraform template generated in step :ref:`Generate an infrastructure template <generate-infra-template>` to create your Role ARN by deploying the template in your AWS account.
+
+As soon as you acquire Role ARN, enter it into the **Role ARN** field in the **Create custom cloud** wizard, and select **Next** to proceed.
+
+.. note::
+   
+   You can park your cloud setup here, save your current configuration as a draft, and resume creating your cloud later.
+
+Assign projects and customer contacts
+'''''''''''''''''''''''''''''''''''''
+
+Continue working in the **Create custom cloud** wizard by taking the following steps:
+
+1. From the **Assign projects** dropdown menu, select projects for which you want your custom cloud to be available.
+2. To add customer contacts, select their roles using the **Role** dropdown menu, and provide email addresses in the **Email** field.
```

Comment:
Please add a comment that we use the contact information in case our support team needs to reach out to fix a technical issue with the custom cloud.